lib/exporters/jdl_exporter.js

```diff
@@ -27,10 +27,12 @@ module.exports = {
  * Writes down the given JDL to a file.
  * @param jdl the JDL to write.
  * @param path the path where the file will be written.
+ * @param {Object}  [options] - Export options.
+ * @param {Boolean} [options.exportNotRegisteredConfigs] - Export configs not registered on definitions.
  */
-function exportToJDL(jdl, path = 'app.jdl') {
+function exportToJDL(jdl, path = 'app.jdl', options) {
   if (!jdl) {
     throw new Error('A JDLObject has to be passed to be exported.');
   }
-  fs.writeFileSync(path, jdl.toString());
+  fs.writeFileSync(path, jdl.toString(options));
```

Comment:
I don't like the fact that we pass options directly to the JDLObject from the exporter, that means there's a coupling between these two (almost independent) parts of JCore.
As I understand the code right now, I think that the toString method takes options with a boolean, might as well have two functions.